### ISSUE TITLE
Expecting Multimple model of single sequence run

### ISSUE THREAD

**Issue by Hossen-sahadot97** [sokrypton/ColabFold#561]
Can I have multiple models (5 models) of single sequence with varing quality?

**Comment by sokrypton** [comment]
Yes. Even within the same model, changing the random seed will drastically result in varying quality.

**Comment by Hossen-sahadot97** [comment]
How to get multiple model in omegafold? There are any option to include random seed number in Omegafold?
